keyboards/dztech/dz68rgb/dz68rgb.c

```diff
@@ -0,0 +1,165 @@
+	{1, C1_16, C2_16, C3_16}, // LD8
+	{1, C9_9,  C8_9,  C7_9},  // LD9
+	{1, C9_10, C8_10, C7_10}, // LD10
+	{1, C9_11, C8_11, C7_11}, // LD11
+	{1, C9_12, C8_12, C7_12}, // LD12
+	//{1, C9_13, C8_13, C7_13}, // LD13
+	{1, C9_14, C8_14, C7_14}, // LD14
+	{1, C9_15, C8_15, C6_14}, // LD15
+	{1, C9_16, C7_15, C6_15}, // LD16
+	{1, C8_16, C7_16, C6_16}, // LD17
+};
+rgb_led g_rgb_leds[DRIVER_LED_TOTAL] = {
+    {{1|(7<<4)},   {16*7,  16}, 4},
+    {{1|(6<<4)},   {16*6,  16}, 4},
+    {{1|(5<<4)},   {16*5,  16}, 4},
+    {{1|(4<<4)},   {16*4,  16}, 4},
+    {{1|(3<<4)},   {16*3,  16}, 4},
+    {{1|(2<<4)},    {16*2,  16}, 4},
+    {{1|(1<<4)},    {16*1,  16}, 4},
+    {{1|(0<<4)},    {16*0,  16}, 1},
+	{{2|(0<<4)},    {16*0,  32}, 1},
+    {{0|(8<<4)},    {16*8,  0}, 1},
+    {{0|(7<<4)},    {16*7,  0}, 1},
+    {{0|(6<<4)},    {16*6,   0}, 1},
+    {{0|(5<<4)},    {16*5,  0}, 1},
+    {{0|(4<<4)},    {16*4,  0}, 1},
+    {{0|(3<<4)},    {16*3, 0}, 1},
+    {{0|(2<<4)},    {16*2,  0}, 1},
+    {{0|(1<<4)},    {16*1,  0}, 1},
+    {{0|(0<<4)},    {16*0,  0}, 1},
+    {{0|(9<<4)},    {16*9,  0}, 1},
+    {{0|(10<<4)},   {16*10,  0}, 1},
+    {{0|(11<<4)},   {16*11,  0}, 1},
+    {{0|(12<<4)},   {16*12,  0}, 1},
+    {{0|(13<<4)},   {16*13,  0}, 1},
+    {{0|(14<<4)},   {16*14,  0}, 1},
+    {{1|(14<<4)},    {16*14,  16}, 1},
+    {{2|(14<<4)},    {16*14,  32}, 1},
+    {{1|(8<<4)},    {16*8,  16}, 4},
+    {{1|(9<<4)},    {16*9,  16}, 4},
+    {{1|(10<<4)},    {16*10,  16}, 4},
+    {{1|(11<<4)},    {16*11,  16}, 4},
+    {{1|(12<<4)},    {16*12,  16}, 4},
+    {{1|(13<<4)},    {16*13,  16}, 4},
+    {{3|(14<<4)},    {16*14,  48}, 1},
+    {{4|(14<<4)},    {16*14,  64}, 1},
+    {{4|(13<<4)},    {16*13,  64}, 1},
+	{{4|(5<<4)},   {16*5,  64}, 1},
+	{{3|(5<<4)},   {16*5,  48}, 4},
+    {{3|(4<<4)},   {16*4,  48}, 4},
+    {{3|(3<<4)},   {16*3,  48}, 4},
+    {{3|(2<<4)},   {16*2,  48}, 4},
+    {{3|(1<<4)},   {16*1,  48}, 4},
+    {{4|(2<<4)},   {16*2,  64}, 1},
+    {{4|(1<<4)},   {16*1,  64}, 1},
+    {{2|(6<<4)},   {16*6, 32}, 4},
+    {{2|(5<<4)},   {16*5, 32}, 4},
+    {{2|(4<<4)},   {16*4, 32}, 4},
+    {{2|(3<<4)},   {16*3, 32}, 4},
+    {{2|(2<<4)},   {16*2, 32}, 4},
+    {{2|(1<<4)},   {16*1, 32}, 4},
+    {{3|(0<<4)},   {16*0,  48}, 1},
+    {{4|(0<<4)},   {16*0,  64}, 1},
+	{{2|(7<<4)},   {16*7,  32}, 4},
+	{{2|(8<<4)},   {16*8,  32}, 4},
+    {{2|(9<<4)},   {16*9,  32}, 4},
+    {{2|(10<<4)},   {16*10,  32}, 4},
+    {{2|(11<<4)},   {16*11,  32}, 4},
+    {{2|(13<<4)},   {16*13,  32}, 4},
+    {{3|(10<<4)},   {16*10,  48}, 4},
+    {{3|(11<<4)},   {16*11,  48}, 4},
+    {{3|(13<<4)},   {16*13,  48}, 4},
+    {{3|(6<<4)},   {16*6,  48}, 4},
+    {{3|(7<<4)},   {16*7,  48}, 4},
+    {{3|(8<<4)},   {16*8,  48}, 4},
+    {{3|(9<<4)},   {16*9,  48}, 4},
+    {{4|(8<<4)},   {16*8,  64}, 1},
+    {{4|(9<<4)},   {16*9,  64}, 1},
+    {{4|(10<<4)},  {16*10,  64}, 1},
+    {{4|(11<<4)},  {16*11,  64}, 1},
+};
+void matrix_init_kb(void) {
```

Comment:
These functions are set to the default, so they aren't actually needed here.

The only one that would be, are the suspend functions